### PR TITLE
Webpack setup

### PR DESCRIPTION
**Pull request by jwu910**
Alex, this should set up your webpack configs and dependencies. Had to adjust a little bit of the source to make sure it built but my changes should be revisited. Not familiar enough wit hteh Atomic Design concepts and how it exposes components as an importable ojbect. If we can get rid of the relative paths it would def look cleaner.

TODO: Currently dev server port just displays file system. Need to figure out how to have dev server serve the right files.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,24 +4,9 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
-    <!--
-      manifest.json provides metadata used when your web app is added to the
-      homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
```
Comment (alexandervalencia) [inline public/index.html]: These shouldn't have been removed

**Comment by alexandervalencia** [comment]
Hey Josh, I'd like to try taking this approach rather than using `eject`: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables

If we can't do what we want, we can take the `eject` route then.

**Comment by jwu910** [comment]
Sure. we can close this PR and shelve the work :+1:

**Comment by alexandervalencia** [comment]
Thanks, sorry about the (potentially) lost effort

**Comment by jwu910** [comment]
:100:

**Pull request closed without merging**